src/module-vsbridge-indexer-catalog/Model/ResourceModel/Product/Bundle.php

```diff
@@ -134,9 +153,9 @@ private function initOptions($storeId)
      *
      * @return void
      */
-    private function initSelection()
+    private function initSelection($storeId)
```

Comment:
Please change to `private function initSelection(int $storeId)`  and update phpDoc for function.